From a042fb94e7e3a0dca547dabfec4bf4d1b482a5e0 Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Fri, 16 Aug 2013 06:43:07 +0400
Subject: [PATCH 2/3] ARM: dts: imx6qdl-wandboard: add bluetooth control lines
 for bcm4329

This patch sets the required pins to enable the bluetooth module.

Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>
---
 arch/arm/boot/dts/imx6qdl-wandboard.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
index cc042ea..1ab3d62 100644
--- a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
@@ -95,6 +95,9 @@
 				MX6QDL_PAD_ENET_RXD1__GPIO1_IO26 0x80000000 /* WL_REG_ON */
 				MX6QDL_PAD_ENET_TXD1__GPIO1_IO29 0x80000000 /* WL_HOST_WAKE */
 				MX6QDL_PAD_ENET_TXD0__GPIO1_IO30 0x80000000 /* WL_WAKE */
+				MX6QDL_PAD_EIM_DA13__GPIO3_IO13  0x80000000 /* BT_ON */
+				MX6QDL_PAD_EIM_DA14__GPIO3_IO14  0x80000000 /* BT_WAKE */
+				MX6QDL_PAD_EIM_DA15__GPIO3_IO15  0x80000000 /* BT_HOST_WAKE */
 			>;
 		};
 	};
-- 
1.8.1.2

